tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java

```diff
@@ -176,15 +189,17 @@ public void start() throws Exception {
 
         // start external services
         Map<String, GenericContainer<?>> externalServices = spec.externalServices;
-        if (null != externalServices) {
-            externalServices.entrySet().forEach(service -> {
+        if (externalServices != null) {
```

Comment:
I would prefer not to mix style changes with functionality changes.

Reply:
we are adding withnetwork and withNetworkAliases to make external services work.